keyboards/helix/helix.c

```diff
@@ -0,0 +1,153 @@
+/* Copyright 2020 yushakobo
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "helix.h"
+
+bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
```

Comment:
Honestly, I'd prefer that the user callback go up here, as: 
```suggestion
bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
    if (!process_record_user(keycode, record)) { 
        return false;
    }
```

Mostly because this allows it to be overwritten by users, if need.  But honestly, I'm not sure if that is really needed here.

Reply:
I kept it this way because all the default keymaps utilise these keycodes, but I suppose the functionality would still be there given there's no `_user` function overriding them.